data/language/en-GB.txt

```diff
@@ -4321,6 +4321,8 @@ STR_6009    :{SMALLFONT}{BLACK}Select cut height
 STR_6010    :{COMMA2DP32}m
 STR_6011    :{COMMA1DP16}ft
 STR_6012    :{COMMA1DP16}
+STR_6013    :Rides don't decrease in value over time
+STR_6014    :{SMALLFONT}{BLACK}The value of a ride won't decrease over time, so guests will not suddenly think that a ride is too expensive.
```

Comment:
I think this should be reworded to something such as "Rides will not age over time and stay 'new', preventing thoughts such as high cost."